nette/notejam/app/presenters/templates/Note/default.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p class="hidden-text">Last edited {$note->updated_at|date:'d. M. Y'}</p>
+	<div class="note"><p>{$note->text}</p>
+	</div>
+	<a n:href="Note:edit $note->id" class="button">Edit</a>
```

Comment:
I think we should be consistent and use the `,` between arguments `n:href="Note:edit, $note->id"`